src/Tribe/Status/Abstract_Commerce.php

```diff
@@ -129,44 +129,43 @@ public function remove_line_total( $value ) {
 	 *
 	 * @since 4.10
 	 *
-	 * @param $ticket_sold object an object of the ticket to get counts
-	 * @param $post_id int an ID of the post the ticket is attached to
+	 * @param array $ticket_sold An object of the ticket to get counts.
+	 * @param int   $post_id     An ID of the post the ticket is attached to.
 	 *
 	 * @return string a string Ticket name, sold, and availability
 	 */
-	public function get_ticket_sale_infomation( $ticket_sold, $event_id ) {
-
+	public function get_ticket_sale_infomation( $ticket_sold, $post_id ) {
 		ob_start();
 		?>
 		<div class="tribe-event-meta tribe-event-meta-tickets-sold-itemized">
 			<?php
-				echo $this->get_name_and_sold_for_ticket( $ticket_sold, $event_id );
+				echo $this->get_name_and_sold_for_ticket( $ticket_sold, $post_id );
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$this'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$this'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$post_id'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$post_id'.

<sub>hash: b159e126e4871228f1021f5199f6f1a0</sub>